website/pages/docs/internals/telemetry.mdx

```diff
@@ -85,11 +85,13 @@ These metrics represent operational aspects of the running Vault instance.
 | `vault.cache.miss`                   | Number of times a value was not in the LRU cache. The results in a read from the configured storage.                                                                                                | cache miss   | counter |
 | `vault.cache.write`                  | Number of times a value was written to the LRU cache.                                                                                                                                               | cache write  | counter |
 | `vault.cache.delete`                 | Number of times a value was deleted from the LRU cache. This does not count cache expirations.                                                                                                      | cache delete | counter |
+| `vault.core.active`                  | Has value 1 when the vault node is active, and 0 when node is in standby.                                                                                                                           | bool | gauge   |
+| `vault.core.activity.fragment_size`  | Number of entities or tokens (depending on the "type" label) observed by the local node.                                                                                                            | tokens | counter  |
+| `vault.core.activity.segment_write`  | Duration of time taking writing activity log segments to storage.                                                                                                                                   | ms   | summary  |
```

Comment:
I might just be misunderstanding, but I think the description sentence could be more clear here. Is it the duration of the time taken to write the seg to storage?

Reply:
taking -> taken